Week1/homework/app.js

```diff
@@ -1,11 +1,93 @@
 'use strict';
 {
-  const bookTitles = [
+  const bookIDs = [
     // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'harry_potter_chamber_secrets',
+    'the_ragged_edge_of_night',
+    'a_case_of_need',
+    'the_ocean_liner',
+    'what_I_have_done',
+    'dirty_headlines',
+    'the_storyteller_s_secret',
+    'origin',
+    'harry_potter_and_the_sorcerer_s_stone',
+    'her_cherry',
+    'worth_the_wait'
   ];
 
+  const booksInfo = {
+    harry_potter_chamber_secrets: { title: 'Harry Potter Chamber Secrets', language: 'Engels', author: 'J. K. Rowling' },
+    the_ragged_edge_of_night: { title: 'The Ragged Edge of Night', language: 'Engels', author: 'Olivia Hawker' },
+    a_case_of_need: { title: 'A Case of Need', language: 'Engels', author: ' Michael Crichton' },
+    the_ocean_liner: { title: 'The Ocean Liner', language: 'Engels', author: 'Marius Gabriel' },
+    what_I_have_done: { title: 'What I\'ve Done', language: 'Engels', author: 'Melinda Leigh' },
+    dirty_headlines: { title: 'Dirty Headlines', language: 'Engels', author: 'L J Shen' },
+    the_storyteller_s_secret: { title: 'The Storyteller\'s Secret', language: 'Engels', author: 'Jini S' },
+    origin: { title: 'Origin', language: 'Engels', author: 'Dan Brown' },
+    harry_potter_and_the_sorcerer_s_stone: { title: 'Harry Potter And The Sorcerer\'s Stone', language: 'Engels', author: 'J.K.Rowling' },
+    her_cherry: { title: 'Her Cherry', language: 'Engels', author: 'Penelope Bloom' },
+    worth_the_wait: { title: 'Worth The Wait', language: 'Engels', author: 'Claudia Connor' }
+  };
 
-  // Replace with your own code
-  console.log(bookTitles);
+  function render(container) {
+    const ulElem = document.createElement('ul');
+
+    for (let book of bookIDs) {
```

Comment:
A single element of the `bookIDs` is a `bookID` (or simply `id`). It is not a `book`.